src/OrchardCore.Modules/OrchardCore.Users/Startup.cs

```diff
@@ -119,6 +119,8 @@ public override void ConfigureServices(IServiceCollection services)
             {
                 o.Cookie.Name = "orchauth_" + _tenantName;
                 o.Cookie.Path = new PathString(_tenantPrefix);
+                // Disabling same-site is required for OpenID's module prompt=none support to work correctly.
+                o.Cookie.SameSite = SameSiteMode.None;
```

Comment:
@sebastienros FYI.

Reply:
No way we could make this optional by adding a service to update auth cookies?

Reply:
We can certainly move that in the OpenID module so that the same site flag of the Users application cookie is overridden only when the OpenID module is enabled, if you prefer.

Reply:
Changed.